core/frontend/src/utils/pattern_validators.ts

```diff
@@ -0,0 +1,37 @@
+export function validate_socket_port(port: number, public_range = true): boolean {
```

Comment:
Wait, are we using CamelCase or snake_case for files on this project ? I can see both types in this patch.

Reply:
Just added a note about style on the Wiki.